src/index.js

```diff
@@ -20,15 +44,58 @@ function createElement(tagName, props, ...children) {
   return element;
 }
 
-function render() {
+function render(showValue, accumulatedValue = 0, lastClicked = '', operator = '') {
+  function handleClick(clickedValue) {
+    /* 숫자 연속 입력 처리 */
+    if (isNumber(lastClicked) && isNumber(clickedValue)) {
+      const newNumber = parseInt(`${lastClicked}${clickedValue}`, 10);
+      render(newNumber, newNumber, newNumber, operator);
+      return;
+    }
+
+    /* 숫자 클릭한 경우 */
+    if (isNumber(clickedValue) && operator === '') {
+      render(clickedValue, clickedValue, clickedValue, operator);
+    } else if (isNumber(clickedValue) && operator !== '') {
+      render(clickedValue, accumulatedValue, clickedValue, operator);
+      return;
+    }
+
+    /* = 클릭한 경우 */
+    if (clickedValue === '=' && operator !== '') {
+      const newAccumulate = calculate(operator, accumulatedValue, lastClicked);
+      render(newAccumulate, newAccumulate, clickedValue, '');
+    } else if (clickedValue === '=') {
+      render(showValue, accumulatedValue, lastClicked, operator);
+      return;
+    }
+
+    /* 연산자 클릭한 경우 */
+    if (!isNumber(clickedValue)) {
+      const newAccumulate = calculate(operator, accumulatedValue, lastClicked);
+      render(newAccumulate, newAccumulate, clickedValue, clickedValue);
+    }
+  }
+
   const element = (
     <div>
       <p>간단 계산기</p>
+      <p>{showValue}</p>
+      <p>
+        {numbers.map((i) => (
+          <button type="button" onClick={() => { handleClick(i); }}>{i}</button>
```

Comment:
```suggestion
          <button type="button" onClick={() => handleClick(i)}>{i}</button>
```

같은 형태로 표현하시면 더 코드가 깔끔해질 것 같아요 ;)

Reply:
아하! 넵 수정하겠습니다✔